lib/oauth2client.js

```diff
@@ -433,7 +445,7 @@ OAuth2Client.prototype._createPostOption = function (postUrl, urlEncodedRequestF
     var postOptions = util.createRequestOptions(
         this,
     {
-            'url' : postUrl,
+            'url' : url.format(postUrl),
```

Comment:
but if no platform actually send down any qp to token endpoint, then we should remove the api-version=1.0, it doesn't need to be there.